### ISSUE TITLE
MatSortHeader doesn't announce asc/desc status on interaction under NVDA

### ISSUE THREAD

**Issue by deepakparamesh** [angular/components#20532]
### Issue Description : 

Speech Narrator (NVDA) is not narrating 'Ascending' and 'Descending' when the column is sorted by clicking the sortable Header.

Steps to reproduce:
1. Install a speech Narrator.
2. Traverse to sortable column header by keeping the speech Narrator ON.
3. click the header.
 
 
#### Expected Behavior

Ideally the narrator should narrate as 'ascending' or 'descending' when they are clicked.

#### Actual Behavior

The narrated is idle when header is clicked.

#### Environment

- Angular: 8.2.14
- CDK/Material: 8.2.3
- Browser(s): Edge Chromium, Google Chrome
- Operating System (e.g. Windows, macOS, Ubuntu): Windows, MacOs, Ubuntu

**Comment by crisbeto** [comment]
We do use `aria-sort` to indicate what the sort order is, although it seems like it only gets announced if the column is sorted when the user lands on it. I'm not sure whether there's any other way we could indicate it when the changes the order themselves. My guess is that our best bet would be to announce something using the `LiveAnnouncer`.

**Comment by jelbourn** [comment]
Confirmed that this is still an issue. As Kristiyan says above, the status is read when you navigate to the sort-header, but nothing is announced upon interaction.

**Comment by zarend** [comment]
Hello @deepakparamesh 

I cannot reproduce, and I believe this has already been fixed. @deepakparamesh does the issue persist for you? That way we know if we need to take action on this?

NVDA Version 2022.3; Windows 10 Enterprise Version 21H2; Firefox 108.0.2 (64-bit); Chrome Version 109.0.5414.75 (Official Build) (64-bit)

Thanks,
Zach

**Comment by zarend** [comment]
Hello,

Thank you for reporting this issue. Unfortunately we are not able to take action if we cannot reproduce it. This might have already been fixed. If there are still problems, please provide reproduction steps or file a new issue report.

- Zach
